mipengine/node/tasks/views.py

```diff
@@ -74,8 +75,8 @@ def create_data_model_views(
 
     Returns
     ------
-    List[str]
-        The names of the created views
+    List[str(TableInfo)]
```

Comment:
should also change the return type annotation

Reply:
It's still a string, since it's serialized.